src/Raven.Server/Documents/Handlers/StatsHandler.cs

```diff
@@ -29,6 +29,12 @@ public Task Stats()
 
                 var stats = new DatabaseStatistics();
                 stats.LastDocEtag = DocumentsStorage.ReadLastDocumentEtag(context.Transaction.InnerTransaction);
+
+                if (stats.LastDocEtag != null)
+                {
+                    stats.LastChangeVector = Database.DocumentsStorage.GetDocumentsFrom(context, stats.LastDocEtag.Value).FirstOrDefault()?.ChangeVector;
```

Comment:
We already are tracking the global change vector, yes that.
Note that there is a major difference between what you are doing here and the actual observed global change vector